st/clitests/rollback_spec.py

```diff
@@ -308,7 +308,6 @@ def clean_18mb_multipart():
 S3cmdTest('s3cmd can create bucket').create_bucket("seagatebucket").execute_test().command_is_successful()
 # ************ Put bucket versioning  TEST ************
 S3fiTest('s3cmd enable FI PUT KV').enable_fi("enable", "always", "motr_kv_put_fail").execute_test().command_is_successful()
```

Comment:
I think all these lines can be delete, but let's follow up later and not worry about it now.

Reply:
created ticket for follow up : https://jts.seagate.com/browse/EOS-26533
Please resolve this issue